src/testRunner/unittests/tsbuild/helpers.ts

```diff
@@ -270,11 +270,12 @@ interface Symbol {
         tick: () => void;
         baseFs: vfs.FileSystem;
         newSys: TscCompileSystem;
+        cleanBuildDescripencies: TscIncremental["cleanBuildDescripencies"];
     }
     function verifyIncrementalCorrectness(input: () => VerifyIncrementalCorrectness, index: number) {
         it(`Verify emit output file text is same when built clean for incremental scenario at:: ${index}`, () => {
             const {
-                scenario, subScenario, commandLineArgs,
+                scenario, subScenario, commandLineArgs, cleanBuildDescripencies,

```

Comment:
huge thanks for this fix!!

do you mean Discrepancies? it looks like that misspelling may be present several places